pkg/utils/keyutil/util.go

```diff
@@ -41,6 +41,36 @@ func MinKey(a, b []byte) []byte {
 	return a
 }
 
+// MaxKeyWithBoundary returns the bigger key for the given keys.
+// it is different from MaxKey, if the key is empty and the boundary is right, the empty key is biggest,
+func MaxKeyWithBoundary(a, b []byte, opt boundary) []byte {
```

Comment:
How about changing to MaxStartKey/MinEndKey? The boundary is useless.

Reply:
good idea